tables/table_json_key_value.go

```diff
@@ -0,0 +1,82 @@
+package config
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+	"gopkg.in/yaml.v3"
+)
+
+func tableJSONKeyValue(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "json_key_value",
+		Description: "List all key value pairs from given JSON file.",
+		List: &plugin.ListConfig{
+			Hydrate: listJSONKeyValue,
+			KeyColumns: plugin.KeyColumnSlice{
+				{
+					Name:    "path",
+					Require: plugin.Optional,
+				},
+			},
+		},
+		Columns: []*plugin.Column{
+			{Name: "path", Type: proto.ColumnType_STRING, Description: "Specifies the path of the JSON file."},
+			{Name: "key_path", Type: proto.ColumnType_LTREE, Transform: transform.FromField("Key").Transform(keysToSnakeCase), Description: "Specifies full path of a key in JSON file."},
+			{Name: "value", Type: proto.ColumnType_STRING, Description: "Specifies the value of the corresponding key."},
+			{Name: "keys", Type: proto.ColumnType_JSON, Transform: transform.FromField("Key"), Description: "The array representation of path of a key."},
+			{Name: "tag", Type: proto.ColumnType_STRING, Description: "Specifies the data type of the value."},
```

Comment:
What is a JSON `tag`? I'm assuming this is the value type, but is this a common term in JSON?